aait/backend/group-2A/Application/Features/Comment/Handlers/Queries/GetCommentRequestHandler.cs

```diff
@@ -22,7 +24,7 @@ public async Task<CommentDto> Handle(GetCommentRequest request, CancellationToke
         var comment = await _unitOfWork.commentRepository.Get(request.commentId);
         if (comment == null)
         {
-            throw new Exception("Could not find comment");
+            throw new NotFoundException(nameof(Domain.Entities.Comment), request.commentId);
         }
 
         return _mapper.Map<CommentDto>(comment);
```

Comment:
instead of returning the comment object . you can return the response object you have defined